scripts/run_tests_functional.sh

```diff
@@ -34,7 +34,3 @@ yarn test_functional
 
 display_result $? 5 "Front end functional tests"
 
-echo "Running census functional tests"
-yarn test_census
```

Comment:
May as well remove it from the README aswell.

Reply:
I've just changed it to test_cypress for the moment, since the functionality is still there.